src/main/java/base/Figure.java

```diff
@@ -0,0 +1,19 @@
+package base;
+
+public class Figure implements AreaCalculable, Drawable {
```

Comment:
Since `Figure` is an interface implementation, it should not contain any state (fields like `color`). Instead, consider making this an abstract class if you want to share common fields or methods, or ensure that state is managed in the concrete implementations of the `Figure`.